src/actions/customToast.tsx

```diff
@@ -1,38 +1,23 @@
-import { toast } from 'react-toastify';
+import { toast, ToastOptions } from 'react-toastify';
 
 type type = 'default' | 'info' | 'success' | 'error';
 
+const defaultOptions: ToastOptions | undefined = {
```

Comment:
Why | undefined part is necessarry here??

Reply:
It's my mistake, I will change it next time.